extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,212 @@
+# KHR\_materials\_transmission <!-- omit in toc -->
+
+## Contributors <!-- omit in toc -->
+
+* Mike Bond, Adobe, [@miibond](https://github.com/MiiBond)
+
+## Status <!-- omit in toc -->
+
+Work in Progress
+
+## Dependencies <!-- omit in toc -->
+
+Written against the glTF 2.0 spec.
+
+## Table of Contents <!-- omit in toc -->
+- [Overview](#overview)
+- [Limitations of Alpha as Coverage](#limitations-of-alpha-as-coverage)
+- [Extending Materials](#extending-materials)
+- [Properties](#properties)
+  - [transmissionFactor](#transmissionfactor)
+  - [transmissionTexture](#transmissiontexture)
+- [Tinting](#tinting)
+- [Refraction](#refraction)
+- [Blend Mode](#blend-mode)
+- [Transparent Metals](#transparent-metals)
+- [Transmission BTDF](#transmission-btdf)
+- [Implementation Notes](#implementation-notes)
+- [Schema](#schema)
+- [Reference](#reference)
+- [Appendix: Full Khronos Copyright Statement](#appendix-full-khronos-copyright-statement)
+
+## Overview
+
+Many optically transparent materials are impossible to represent in a physically plausible manner with the core glTF 2.0 PBR material. This is because the core specification only includes the concept of "alpha as coverage" (exposed via the alpha channel of `baseColorFactor` and `baseColorTexture`). Many common materials like glass and plastic require significantly different rendering techniques.
+
+## Limitations of Alpha as Coverage
+Alpha-as-coverage can basically be thought of as describing whether the surface exists or not. When alpha is 0, nothing is rendered while alpha=1 means that the material is solid. It's most useful for materials such as gauze or burlap that only partially cover the area in which the material is rendered. These materials contain lots of small holes where the light can pass through so, strictly speaking, the light isn't actually entering the material itself (i.e. the surface isn't there). For other materials, like clear glass, a photon of light might actually enter the surface and be transmitted through. In this case, the light can be reflected, refracted, absorbed or scattered. Simple alpha-as-coverage cannot represent any of these effects and so we need very different logic to render them correctly.
+
+The most important of these features is surely reflection. Even a completely transparent, infinitely thin, glass material is still visible due to its reflection of light. However, a material using purely alpha as coverage is not visible at all when alpha = 0 (because the surface effectively doesn't exist).
+
+<figure>
+<img src="./figures/OpacityComparison.png"/>
```

Comment:
I just noticed that there appears to be some refraction in this reference image.  Does this need to be updated?

Reply:
Updated. Thanks for noticing that.